v2/emailpassword/custom-ui/init/user-management-dashboard/setup.mdx

```diff
@@ -116,6 +116,10 @@ The user management dashboard is served by the backend SDK, you have to use your
 
 Navigate to `^{form_apiDomain}^{form_apiBasePath}/dashboard` to view the dashboard.
 
+:::note
+If you are using Next.js, upon integrating our backend SDK into your Next.js API folder, the dashboard will be accessible by default at `/api/auth/dashboard`. For frameworks other than Next.js, it can be accessed at `/auth/dashboard`. Should you have customized the `apiBasePath` configuration property, simply navigate to `^{form_apiDomain}^{form_apiBasePath}/dashboard` to access the dashboard.
```

Comment:
```suggestion
If you are using Next.js, upon integrating our backend SDK into your Next.js API folder, the dashboard will be accessible by default at `/api/auth/dashboard`. For frameworks other than Next.js, it can be accessed at `^{form_apiDomain}^/auth/dashboard`. Should you have customized the `apiBasePath` configuration property, simply navigate to `^{form_apiDomain}^{form_apiBasePath}/dashboard` to access the dashboard.
```

Reply:
@mawentowski you also need to add this to all the other files.